streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2475,27 +2501,27 @@ public void shouldSupportKeyChangeKTableFromKStream() {
                 "    Processor: KSTREAM-FILTER-0000000005 (stores: [])\n" +
                 "      --> KSTREAM-SINK-0000000004\n" +
                 "      <-- KSTREAM-MAP-0000000001\n" +
-                "    Sink: KSTREAM-SINK-0000000004 (topic: KSTREAM-TOTABLE-0000000003-repartition)\n" +
+                "    Sink: KSTREAM-SINK-0000000004 (topic: KSTREAM-TOTABLE-0000000002-repartition)\n" +
```

Comment:
Due to the refactoring of the `KStreamImple#toTable()` overloads, we get different names -- but as nothing is released yet, there is backward compatibility concern.